production/helm/promtail/templates/daemonset.yaml

```
@@ -60,10 +60,13 @@ spec:
             {{- toYaml . | nindent 12 }}
             {{- end }}
           env:
-          - name: HOSTNAME
-            valueFrom:
-              fieldRef:
-                fieldPath: spec.nodeName
+            {{- with .Values.env }}
+              {{- toYaml .Values.env | nindent 12 }}
```

```suggestion
              {{- toYaml . | nindent 12 }}
```
This needs to be changed because `with` changes the scope of the block which means `.` points to whatever is followed by `with`.